tests/Generator/resource/source_typeschema.json

```diff
@@ -243,6 +243,9 @@
           "description": "Optional describes the format of the string. Supported are the following types: date, date-time and time. A code generator may use a fitting data type to represent such a format, if not supported it should fall back to a string.",
           "type": "string"
         }
+      },
+      "x-psx-mapping": {
+        "format": "format"
```

Comment:
This has the wrong format, it should be a map with string values but in general we could also remove this